04-onion-routing.md

```diff
@@ -269,7 +272,81 @@ The writer:
 The reader:
   - MUST return an error if `amt_to_forward` or `outgoing_cltv_value` are not present.
 
-The requirements for the contents of these fields are specified [above](#legacy-hop_data-payload-format).
+The requirements for the contents of these fields are specified [above](#legacy-hop_data-payload-format)
+and [below](#basic-multi-part-payments).
+
+### Basic Multi-Part Payments
+
+If the final node receives an onion packet with the `basic_mpp` field,
+then the payment MAY be a "base" atomic multipath payment.  Such
+"base" atomic multipath payments will use the same `payment_hash` for
+all paths.
+
+The `amt_to_forward` value will be the amount for this partial payment
+only.  The `basic_mpp` flag is a promise by the ultimate sender that
+the rest of the payment will follow in succeeding HTLCs; we call these
+HTLCs, which have the same preimage, an "HTLC set".
+
+#### Requirements
+
+The writer:
```

Comment:
nit: Not introduced here but I found the writer/reader wording a bit weird outside bolt-11 shouldn't we stick with sending node/receiving node?

Reply:
These are routed through multiple nodes though, so it's best to be clear.

Reply:
IIRC, we used "processing node" in other parts of the spec to name intermediary nodes